v2/cli/term/event_page.go

```diff
@@ -36,12 +36,13 @@ func newEventPage(
 		workerInfo: tview.NewTextView().SetDynamicColors(true),
 		jobsTable:  tview.NewTable().SetSelectable(true, false),
 		usage: tview.NewTextView().SetDynamicColors(true).SetText(
-			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
+			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](L) [white]Logs    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
 		),
 	}
 	e.eventInfo.SetBorder(true).SetBorderColor(tcell.ColorWhite)
-	e.workerInfo.SetBorder(true).SetTitle(" Worker ")
-	e.jobsTable.SetBorder(true).SetTitle(" Jobs ")
+	e.workerInfo.SetBorder(true).SetTitle("Worker")
+	e.jobsTable.SetBorder(true).SetTitle("Jobs")
```

Comment:
The extra padding was deliberate.

Reply:
Which padding did you mean? I removed the `logModal` since that is defined inside `log_page.go` now

Reply:
You removed extra spaces I placed around the titles.

Reply:
Oops, must have missed that during the rebase. Fixing that now